lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
I don't think so, I would expect it to take the defaults from the blank template.  @metobet Can you test this?